packages/sui-deploy/src/client/now.js

```diff
@@ -7,14 +7,19 @@ const DEFAULT_DIR = join(process.cwd(), 'public')
 const NOW_VERSION = '11.2.8'
 
 // Get args of `now` command according to params
-const getNowCommandArgs = ({name, dir, auth, token}) => {
+const getNowCommandArgs = ({name, dir, auth, token, environmentVars}) => {
   const args = [dir, '--name=' + name, `-t "${token}"`]
   if (auth) {
     const [user, password] = auth.split(':')
     user &&
       password &&
       args.push(`-e SERVE_USER='${user}'`, `-e SERVE_PASSWORD='${password}'`)
   }
+  if (environmentVars) {
+    environmentVars.forEach(ENVIRONMENT_VAR =>
```

Comment:
we usually use uppercase for variables that are hardcoded

Reply:
But this ones are not hardcoded per se am I right? This ones are introduced by the user from prompt.

Reply:
🤔 but it can be anything